backendapi/views.py

```diff
@@ -18,3 +23,83 @@ def scanned(request, tag: bytes):
 
     except Attendant.DoesNotExist:
         return JsonResponse({"error": "No such tag", "valid": False}, status=404)
+
+
+@api_view(["GET", "POST", "PUT", "PATCH", "DELETE"])
+def api_get(request, pk=None):
```

Comment:
Right now, to get started on the frontend, it would make sense to just return some simple test data (hardcoded Response data, possibly a few mutations based on the provided input variable)